src/open_company_web/components/headcount.cljs

```diff
@@ -14,44 +15,128 @@
+      (om/set-state! owner :pt-employees (utils/thousands-separator pt-employees))
+      (om/set-state! owner :contractors (utils/thousands-separator contractors))))
   (render [_]
-    (let [founders (:founders data)
-          executives (:executives data)
-          ft-employees (:ft-employees data)
-          ft-contractors (:ft-contractors data)
-          pt-employees (:pt-employees data)
-          pt-contractors (:pt-contractors data)
-          total-headcount (+ founders executives ft-employees ft-contractors pt-employees pt-contractors)
-          full-time-equivalent (+ founders executives ft-employees ft-contractors (/ pt-employees 2) (/ pt-contractors 2))]
-      (r/well {:class "report-list headcount clearfix"}
-        (dom/div {:class "report-list-left"}
-          (om/build report-editable-line {:cursor data :key :founders :label "founder"})
-          (om/build report-editable-line {:cursor data :key :executives :label "executive"})
-          (om/build report-editable-line {:cursor data :key :ft-employees :label "full-time employee"})
-          (om/build report-editable-line {:cursor data :key :pt-employees :label "part-time employee"})
-          (om/build report-editable-line {:cursor data :key :ft-contractors :label "full-time contractor"})
-          (om/build report-editable-line {:cursor data :key :pt-contractors :label "part-time contractor"})
-          (dom/div
-            (om/build report-line {:number total-headcount :label "total" :pluralize false})
-            (dom/span ", ")
-            (om/build report-line {:number full-time-equivalent :label "full-time equivalent"}))
-          (om/build comment-component {:cursor data :key :comment}))
-        (om/build pie-chart (get-chart-data data))))))
+    (let [founders (if (contains? data :founders) (:founders data) 0)
+          executives (if (contains? data :executives) (:executives data) 0)
+          ft-employees (if (contains? data :ft-employees) (:ft-employees data) 0)
+          pt-employees (if (contains? data :pt-employees) (:pt-employees data) 0)
+          contractors (if (contains? data :contractors) (:contractors data) 0)
+          total (+ founders executives ft-employees pt-employees contractors)]
+      (p/panel {:header (dom/h3 "Headcount") :class "headcount clearfix"}
+        (dom/div {:class "headcount row"}
+          (dom/form {:class "form-horizontal col-md-6"}
+            (dom/div {:class "form-group"}
+              (dom/label {:for "founders" :class "col-md-4 control-label"} "Founders")
+              (dom/div {:class "input-group col-md-2"}
+                (dom/input {
+                  :type "text"
+                  :class "form-control"
+                  :value (om/get-state owner :founders)
+                  :on-change #(om/set-state! owner :founders (.. % -target -value))
+                  :on-focus #(om/set-state! owner :founders founders)
+                  :on-blur (fn [e]
```

Comment:
@bago2k4 the comments marking DOM sections like you did in the other views are good. Should add them here to headcount. Same comments apply about breaking this function down, and DRYing if possible.